frontend/src/components/tabs/RuntimeNodeDetailsV2.test.tsx

```diff
@@ -23,6 +23,18 @@ import { NodeMlmdInfo } from 'src/pages/RunDetailsV2';
 import { RuntimeNodeDetailsV2 } from 'src/components/tabs/RuntimeNodeDetailsV2';
 import { Execution, Value } from 'src/third_party/mlmd';
 import TestUtils from 'src/TestUtils';
+import fs from 'fs';
+import jsyaml from 'js-yaml';
+
+const v2PVCPipelineSpecPath = 'src/data/test/create_mount_delete_dynamic_pvc.yaml';
+const v2PVCYamlString = fs.readFileSync(v2PVCPipelineSpecPath, 'utf8');
+// The templateStr used in RuntimeNodeDetailsV2 is not directly from yaml file.
+// Instead, it is from BE (already been processed).
+const v2PVCTemplateStringObj = {
+  pipeline_spec: jsyaml.safeLoadAll(v2PVCYamlString)[0],
+  platform_spec: jsyaml.safeLoadAll(v2PVCYamlString)[1],
+};
+const v2PVCTemplateString = jsyaml.safeDump(v2PVCTemplateStringObj);
```

Comment:
Use `all uppercase letters, with words separated by underscores` to name consts.